CodeStarterCamp_Week2/Week2Step3.swift

```diff
@@ -5,22 +5,27 @@
 //  Created by 홍세희 on 2023/08/29.
 //
 
-var allLottoNumberStorage: Dictionary<String , Set<Int>> = [String : Set<Int>]()
+var lottoNumberStorage: [String : Array<Int>] = [:]
```

Comment:
Array의 경우도 축약형으로 해주시면 좋겠네요~~
```suggestion
var lottoNumbersStorage: [String: [Int]] = [:]
```